.github/workflows/tests.yml

```diff
@@ -56,14 +56,21 @@ jobs:
 
       - uses: actions/checkout@v4
 
-      - name: Setup FFmpeg
-        uses: Iamshankhadeep/setup-ffmpeg@v1.1
+      - if: matrix.os == 'ubuntu-latest'
+        name: Setup FFmpeg
+        uses: Iamshankhadeep/setup-ffmpeg@v1.1  # does not support arm64
         with:
           # Not strictly necessary, but it may prevent rate limit
           # errors especially on GitHub-hosted macos machines.
           token: ${{ secrets.GITHUB_TOKEN }}
           version: "4.4"
 
+      - if: matrix.os == 'macos-latest'
+        name: Setup FFmpeg
+        run: |
+          brew install ffmpeg@4
```

Comment:
Would pinning more strictly to 4.4 maybe help with the problem where it's producing different results?

Reply:
I'd done pretty extensive testing using the same version, same python previously and still gotten different results

https://github.com/drivendataorg/zamba/issues/252